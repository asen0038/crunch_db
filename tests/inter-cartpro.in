set a 2 2 4 4 8 0
set b 4 9 0 2
set c 2 7 7 7
inter a b c
cartprod a b c
list entries
bye
