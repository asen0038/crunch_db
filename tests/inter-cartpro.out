>
ok

>
ok

>
ok

>
[2]

>
[ [2 4 2] [2 4 7] [2 4 7] [2 4 7] [2 9 2] [2 9 7] [2 9 7] [2 9 7] [2 0 2] [2 0 7] [2 0 7] [2 0 7] [2 2 2] [2 2 7] [2 2 7] [2 2 7] [2 4 2]
[2 4 7] [2 4 7] [2 4 7] [2 9 2] [2 9 7] [2 9 7] [2 9 7] [2 0 2] [2 0 7] [2 0 7] [2 0 7] [2 2 2] [2 2 7] [2 2 7] [2 2 7] [4 4 2] [4 4 7]
[4 4 7] [4 4 7] [4 9 2] [4 9 7] [4 9 7] [4 9 7] [4 0 2] [4 0 7] [4 0 7] [4 0 7] [4 2 2] [4 2 7] [4 2 7] [4 2 7] [4 4 2] [4 4 7] [4 4 7] [
4 4 7] [4 9 2] [4 9 7] [4 9 7] [4 9 7] [4 0 2] [4 0 7] [4 0 7] [4 0 7] [4 2 2] [4 2 7] [4 2 7] [4 2 7] [8 4 2] [8 4 7] [8 4 7] [8 4 7] [8
9 2] [8 9 7] [8 9 7] [8 9 7] [8 0 2] [8 0 7] [8 0 7] [8 0 7] [8 2 2] [8 2 7] [8 2 7] [8 2 7] [0 4 2] [0 4 7] [0 4 7] [0 4 7] [0 9 2] [0
9 7] [0 9 7] [0 9 7] [0 0 2] [0 0 7] [0 0 7] [0 0 7] [0 2 2] [0 2 7] [0 2 7] [0 2 7] ]

>
bye
